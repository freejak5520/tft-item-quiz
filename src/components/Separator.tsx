const Separator = () => {
  return (
    <div className="border-border-300 dark:border-border-700 border-b-2" />
  );
};

export default Separator;
