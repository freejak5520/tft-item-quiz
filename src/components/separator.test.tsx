import { render } from "@testing-library/react";
import Separator from "./separator";

describe("Separator", () => {
  it("renders the separator", () => {
    const { container } = render(<Separator />);

    expect(container).toMatchSnapshot();
  });
});
