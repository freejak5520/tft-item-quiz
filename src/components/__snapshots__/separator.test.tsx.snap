// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator renders the separator 1`] = `
<div>
  <div
    class="border-b-2 border-border-300 dark:border-border-700"
  />
</div>
`;
